topics/assembly/tutorials/vgp_workflow_training/tutorial.md

```diff
@@ -166,61 +243,139 @@ Once we have imported the datasets and the workflows, we can start with the geno
 >
 {: .comment}
 
-# Genome profile analysis
+# Performing the assembly
+
+Workflows listed in [Fig. 1](#vgp-galaxy-pipeline-a-collection-of-versatile-workflows) support a variety of "analysis trajectories". The majority of species that were sequenced by the {VGP} usually contain {HiFi} reads for the individual being sequenced supplemented with {Hi-C} data. As a result most assemblies performed by us follow the trajectory **B**. This is why this tutorial was designed to follow this trajectory as well.  
+
+## Genome profile analysis (WF1)
 
 [{% icon exchange %} Switch to step by step version]({% link topics/assembly/tutorials/vgp_genome_assembly/tutorial.md %}#genome-profile-analysis)
 
-Now that our data and workflows are imported, we can run our first workflow. Before the assembly can be run, we need to collect metrics on the properties of the genome under consideration, such as the expected genome size according to our data. The present pipeline uses **Meryl** for generating the k-mer database and **Genomescope2** for determining genome characteristics based on a k-mer analysis.
+Now that our data and workflows are imported, we can run our first workflow. Before the assembly can be run, we need to collect metrics on the properties of the genome under consideration, such as the expected genome size according to our data. The present pipeline uses **Meryl** for generating the *k*-mer database and **Genomescope2** for determining genome characteristics based on a *k*-mer analysis.
 
-> <hands-on-title>VGP genome profile analysis workflow</hands-on-title>
->
-> 1. Click in the **Workflow** menu, located in the top bar
-> 2. Click in the {% icon workflow-run %} **Run workflow** buttom corresponding to `VGP genome profile analysis`
-> 3. In the **Workflow: VGP genome profile analysis** menu:
->   - {% icon param-collection %} "*Collection of Pacbio Data*": `7: HiFi_collection`
->   - "*K-mer length*": `31`
->   - "*Ploidy*": `2`
-> 4. Click on the <kbd>Run workflow</kbd> buttom
->
-> > <comment-title>K-mer length</comment-title>
-> > In this tutorial, we are using a k-mer length of 31. This can vary, but the VGP pipeline tends to use a k-mer length of 21, which tends to work well for most mammalian-size genomes. There is more discussion about k-mer length trade-offs in the extended VGP pipeline tutorial.
-> {: .comment}
->
-{: .hands_on}
+### **Step 1**: Identify inputs
+
+The profiling workflow takes the following inputs:
+
+1. {HiFi} reads as a collection
+2. *K*-mer length
+3. Ploidy
+
+### **Step 2**: Launch *k*-mer profiling workflow
+
+1. Click in the **Workflow** menu, located in the top bar
+2. Click in the {% icon workflow-run %} **Run workflow** buttom corresponding to `K-mer profiling and QC (WF1)`
+3. In the **Workflow: VGP genome profile analysis** menu:
+   - {% icon param-collection %} "*Collection of Pacbio Data*": `7: HiFi_collection`
+   - "*K-mer length*": `31`
```

Comment:
This seems like another bit of content that should be in a hands-on box, like it previously was.

Reply:
Hands on boxes are very hard to edit and maintain - this is a much clearer way to represent this.